src/Microsoft.Health.Dicom.SqlServer/Features/Workitem/WorkitemSqlQueryGenerator.cs

```diff
@@ -67,8 +67,16 @@ private void AppendFilterTable(string filterAlias)
             StringBuilder
                 .AppendLine("( SELECT ")
                 .Append(VLatest.Workitem.WorkitemUid, WorkitemTableAlias).AppendLine(",")
-                .Append(VLatest.Workitem.WorkitemKey, WorkitemTableAlias).AppendLine()
-                .AppendLine($"FROM {VLatest.Workitem.TableName} {WorkitemTableAlias}");
+                .Append(VLatest.Workitem.WorkitemKey, WorkitemTableAlias).AppendLine();
+
+            if ((int)SchemaVersion >= SchemaVersionConstants.SupportUpsRsWatermarkSchemaVersion)
+            {
+                StringBuilder
+                    .Append(",")
+                    .AppendLine(VLatest.Workitem.Watermark, WorkitemTableAlias);
```

Comment:
should this be `Append(VLatest.Workitem.Watermark, WorkitemTableAlias).AppendLine();`?

Reply:
Done